src/cdk/a11y/live-announcer/live-announcer.spec.ts

```diff
@@ -282,6 +282,15 @@ describe('CdkAriaLive', () => {
     announcer.ngOnDestroy();
   }));
 
+  it('should default politeness to polite', fakeAsync(() => {
+      fixture.componentInstance.content = 'New content';
+      fixture.detectChanges();
+      invokeMutationCallbacks();
+      flush();
```

Comment:
The indentation seems off here.

Reply:
Changed on commit [3021ec5](https://github.com/angular/components/pull/20672/commits/3021ec52bf4dc9d60b1e1e3c7f6d659e32dce1d5)